app/src/main/java/com/simplecity/amp_library/playback/PlaybackManager.java

```diff
@@ -491,6 +492,7 @@ public boolean next(boolean force) {
         }
 
         setQueuePosition(nextPosition);
+        newPlayerEvent(queueManager.getCurrentSong(), PlayerEventType.PLAY);
```

Comment:
Just to check - so this doesn't double-log `PLAY`, right? The `PLAY` event isn't triggered elsewhere when tapping "next"?

Reply:
Yes, I double-checked the database and it looks like it's being logged only once.